internal/resources/ibm/resource_instance.go

```diff
@@ -133,6 +141,13 @@ var ResourceInstanceUsageSchema = []*schema.UsageItem{
 	{Key: "sysdig-secure_VM_NODE_HOUR", DefaultValue: 0, ValueType: schema.Float64},
 	{Key: "aiopenscale_RESOURCE_UNITS", DefaultValue: 1, ValueType: schema.Float64},
 	{Key: "aiopenscale_MODELS_PER_MONTH", DefaultValue: 1, ValueType: schema.Float64},
+	{Key: "dns-svcs_ITEMS", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_MILLION_ITEMS", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_MILLION_ITEMS_CREXTERNALQUERIES", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_NUMBERGLB", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_NUMBERHEALTHCHECK", DefaultValue: 1, ValueType: schema.Int64},
+	{Key: "dns-svcs_NUMBERPOOLS", DefaultValue: 1, ValueType: schema.Int64},
```

Comment:
I believe the cost for pools per hour is similar to GLB instance hours and will need to be calculated in the same way and will need a usage variable for the number of hours for the pool